src/services/userNotification.ts

```diff
@@ -90,3 +90,28 @@ export const fetchSavedSearch = async ({
     path: `saved-searches/key/${key}`,
     options,
   })
+
+export const sendSavedSearchFeedback = async ({
+  reason,
+  key,
+  options = {},
+}: {
+  reason: string
+  key: string
+  options?: ApiCallOptions
+}): Promise<WithValidationError> => {
+  try {
+    await postData({
+      path: `saved-searches/key/${key}/unsubscribe-feedback`,
+      body: { reason },
+      options,
```

Comment:
I'm wondering if we need the authentication here. Can you double check it? Thanks!

Reply:
Checked! We don't need it, we have a UUID in the path: `/key/${key}`